NebulaPatcher/NebulaPlugin.cs

```diff
@@ -82,5 +82,9 @@ void AddNebulaBootstrapper()
 
             Log.Info("Behaviours applied.");
         }
+
+        public string Verson => NebulaModel.Config.ModVersion;
+
+        public bool CheckVersion => true;
```

Comment:
Why is this always fixed to true?

Reply:
That means we want to check version of mod. If its set to false we only check if mod exists on both sides.

Reply:
Typo: Verson => Version (missing i)

Reply:
Oops.

Reply:
I have decided to entirely remove this option. On a second look it just does not make any sense